crs_application/launch/crs.launch.xml

```diff
@@ -16,16 +20,22 @@
 	    <include file="$(find-pkg-share crs_application)/launch/load_robot_model.launch.xml">
 	    </include>
 		
-		<include file="$(find-pkg-share crs_support)/launch/environment_launch.launch.py">
+    <include file="$(find-pkg-share crs_support)/launch/environment_launch.launch.py" unless="$(var at_swri)">
```

Comment:
We should revisit this and pass a directory or file specific to this workcell instead

Reply:
I am not sure what you mean, as in you'd like to pass in a command line argument of the urdf filepath?

Reply:
I was thinking we could pass the path to the urdf file we intended to use as an argument to the environment_launch.launch.py but that might not be possible since this script processes the xacro to create a urdf prior to launching any nodes.

I think it'd be best to create separate **crs_swri.launch.xml** that launches nodes and launch files specific to this workcell

Reply:
That is what the launch file environment_launch_swri.launch.py does. I just made it so that there is a 'swri' param that defaults to false for the crs.launch.xml.